Formula/w/whatweb.rb

```diff
@@ -0,0 +1,38 @@
+class Whatweb < Formula
+  desc "Web scanner. Identify the technology stack that powers a website"
+  homepage "https://morningstarsecurity.com/research/whatweb"
+  url "https://github.com/urbanadventurer/WhatWeb/archive/refs/tags/v0.5.5.tar.gz"
+  sha256 "96dedb6a377184fb8f5fd3f2a81c26ff8c92c4dc1503ce409793a1e7ab23695d"
+  license "GPL-2.0-only"
+  head "https://github.com/urbanadventurer/WhatWeb.git", branch: "master"
+
+  depends_on "rbenv"
+  depends_on "ruby-build"
+  uses_from_macos "ruby" => :build
+
+  # newer ruby version is needed for older macos
+  on_monterey :or_older do
+    depends_on "ruby" => :build
+  end
+
+  def install
+    ENV["GEM_HOME"] = buildpath/"gem_home"
+    system "gem", "install", "bundler"
+    ENV.prepend_path "PATH", buildpath/"gem_home/bin"
+    system "bundle", "install"
+    system "make", "install",
+      "prefix=#{prefix}",
+      "-e", "BINPATH=#{bin}",
+      "-e", "DOCPATH=#{doc}",
+      "-e", "LIBPATH=#{lib}",
+      "-e", "MANPATH=#{man}"
+    system "gem", "install", "bson"
+    system "gem", "install", "bson_ext"
+    system "gem", "install", "mongo"
+    system "gem", "install", "rchardet"
```

Comment:
These should be checksummed resources, not downloaded at random like this.

Reply:
added checksummed resources

Reply:
@oregonpillow @SMillerDev if there's a `Gemfile.lock` I think it's reasonable to use that instead, otherwise: yes, I agree.